### PR TITLE
add gitlab webhook faq

### PR DESCRIPTION
**Pull request by mirnawong1**
this PR adds the following:

- new faq page explaining that a webhook is auto-registered when connect dbt cloud to gitlab. helps with troubleshooting if user can't trigger ci job. 
- new faq page to consolidate ['troubleshooting'](https://docs.getdbt.com/docs/cloud/git/connect-gitlab#troubleshooting) and faq' section in the gitlab set up doc. 
- turned troubleshooting content into faq and removed 'troubleshooting' header.

Screenshot 2024-12-10 at 11 42 59

---
🚀 Deployment available! Here are the direct links to the updated files:

- https://docs-getdbt-com-git-add-gitlab-webhook-info-dbt-labs.vercel.app/docs/cloud/git/connect-gitlab
- https://docs-getdbt-com-git-add-gitlab-webhook-info-dbt-labs.vercel.app/docs/deploy/ci-jobs
- https://docs-getdbt-com-git-add-gitlab-webhook-info-dbt-labs.vercel.app/faqs/Troubleshooting/error-importing-repo
- https://docs-getdbt-com-git-add-gitlab-webhook-info-dbt-labs.vercel.app/faqs/Troubleshooting/gitlab-webhook

### CODE REVIEW

File: website/docs/faqs/Troubleshooting/gitlab-webhook.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: Unable to trigger a CI job with GitLab
```
Comment (mirnawong1) [inline website/docs/faqs/Troubleshooting/gitlab-webhook.md:2]: addresses @Tonayya 's post

File: website/docs/faqs/Troubleshooting/error-importing-repo.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+id: error-importing-repo
+---
+
+If you don't see your repository listed, double-check that:
+- Your repository is in a Gitlab group you have access to. dbt Cloud will not read repos associated with a user.
+
+If you do see your repository listed, but are unable to import the repository successfully, double-check that:
+- You are a maintainer of that repository. Only users with maintainer permissions can set up repository connections.
+
+If you imported a repository using the dbt Cloud native integration with GitLab, you should be able to see the clone strategy is using a `deploy_token`. If it's relying on an SSH key, this means the repository was not set up using the native GitLab integration, but rather using the generic git clone option. The repository must be reconnected in order to get the benefits described above.
```
Comment (nataliefiann) [inline website/docs/faqs/Troubleshooting/error-importing-repo.md]:
```suggestion
If you imported a repository using the dbt Cloud native integration with GitLab, you should be able to see if the clone strategy is using a `deploy_token`. If it's relying on an SSH key, this means the repository was not set up using the native GitLab integration, but rather using the generic git clone option. The repository must be reconnected in order to get the benefits described above.
```

File: website/docs/faqs/Troubleshooting/gitlab-webhook.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+description: "Unable to trigger a CI job"
+sidebar_label: 'Unable to trigger a CI job'
+id: gitlab-webhook
+---
+
+When you connect dbt Cloud to a GitLab repository, GitLab automatically registers a webhook in the background, viewable under the repository settings. This webhook is also used to trigger [CI jobs](/docs/deploy/ci-jobs) when you push to the repository.
+
+If you're unable to trigger a CI job, this usually indicates that the webhook registration is missing or incorrect.
+
+To resolve this issue, navigate to the repository settings in GitLab and view webhook registrations by navigating to GitLab --> **Settings** --> **Webhooks**.
```
Comment (nataliefiann) [inline website/docs/faqs/Troubleshooting/gitlab-webhook.md]:
```suggestion
To resolve this issue, navigate to the repository settings in GitLab and view the webhook registrations by navigating to GitLab --> **Settings** --> **Webhooks**.
```

**Review by nataliefiann — APPROVED** [review]
Hiya @mirnawong1 

Thank you for creating this PR. I've approved this for you with 2 non-blocking suggestions.

Kind Regards
Natalie

**Pull request merged**